news/2022-09-27-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,147 @@
+
+## An insight into the basis of the new rework
+
+The [previous set of changes](https://osu.ppy.sh/home/news/2020-09-15-changes-to-osutaiko-star-rating) made to osu!taiko's performance points & star rating algorithm accounted for several skill areas at once. This made it difficult to collect meaningful feedback on specific skills and led to increased complexity, halting forward development efforts.
+
+This time, we have opted to take a more modular approach by introducing new changes incrementally, in the hopes of being able to fix any reported issues one step at a time. Moving forward, osu!taiko star rating will be receiving continuous updates, placing star rating and performance points in a state of flux, especially over the coming few months.
+
+In the star rating part of this rework, we have focused on "stamina" and "colour" skills, and in the performance points domain, we have made a significant change to how accuracy is calculated and some tweaks to performance point multipliers for specific mods.
+
+Without further ado, let's get into the nitty-gritty of these changes.
+
+## Star rating changes
+
+### The "stamina" skill
+
+#### Issues within the old system
+
+The old system assumed a 2-finger kddk full alternating playstyle (i.e. one in which each subsequent hit is always hit by the other hand regardless of colour). This was implemented as two skills — one for the left hand, and one for the right hand. Instead of using note duration for the split skills, it used the duration between two notes that were pressed with the same hand, a term that we describe as note-pair duration.
+
+In addition to the above, tailored and highly particular nerfs targeted at specific patterns were implemented using code described best as a "cheese detector", which is something that we have since opted to do away with.
+
+The core policy for the all the new reworks under our wing was formed:
+
+**Never assume a specific playstyle or arbitrarily nerf patterns without a strong justification.**
+
+#### The new stamina system
+
+In this section, the keybinds relating to the two colours found within osu!taiko are named k<sup>1</sup>, k<sup>2</sup> and d<sup>1</sup>, d<sup>2</sup>.
+
+Instead of assuming a full-alternate playstyle, we assume that **each note of the same colour is always alternated**. For example, for the pattern kddkddk, we assume the kats will be hit by **k<sup>1</sup>** d<sup>1</sup> d<sup>2</sup> **k<sup>2</sup>** d<sup>1</sup> d<sup>2</sup> **k<sup>1</sup>**, and the dons will be hit by k<sup>1</sup> **d<sup>1</sup> d<sup>2</sup>** k<sup>2</sup> **d<sup>1</sup> d<sup>2</sup>** k<sup>2</sup>.
+
+The reason for this is to assume the best-case scenario for repeated keystrokes regardless of playstyle. In other words, we aim to approximate the theoretical minimum stamina required to play a given pattern, with any theoretically rollable pattern being rolled.
```

Comment:
just going to mention that as a non-taiko player, I have no idea what "rolling" means. But I'm probably not the intended audience for this newspost.

Reply:
My uneducated guess would be pressing three/four keys in a row from left to right or right to left.

Reply:
asked in a taiko discord and got that it's "using the same hand more than once in a row", as opposed to alternating hands.